miner/worker.go

```diff
@@ -852,28 +852,30 @@ LOOP:
 		// Check whether the tx is replay protected. If we're not in the EIP155 hf
 		// phase, start ignoring the sender until we do.
 		if tx.Protected() && !w.chainConfig.IsEIP155(env.header.Number) {
-			//log.Trace("Ignoring reply protected transaction", "hash", tx.Hash(), "eip155", w.chainConfig.EIP155Block)
+			log.Trace("Ignoring reply protected transaction", "hash", tx.Hash(), "eip155", w.chainConfig.EIP155Block)
 			txs.Pop()
 			continue
 		}
 		// Start executing the transaction
 		env.state.Prepare(tx.Hash(), env.tcount)
 
+		from, _ := types.Sender(env.signer, tx)
```

Comment:
Why no error check here?

Reply:
I don't know if we should throw an error from here. I copy/paste what was written on line 851

Reply:
It was commented in line:847, the tx should have been check in the txpool, no they skip the error check here.
```
// Error may be ignored here. The error has already been checked
// during transaction acceptance is the transaction pool.
```